src/spikeinterface/core/unitsaggregationsorting.py

```diff
@@ -77,43 +77,42 @@ def __init__(self, sorting_list, renamed_unit_ids=None):
             if np.all(annotations == annotations[0]):
                 self.set_annotation(annotation_name, sorting_list[0].get_annotation(annotation_name))
 
-        property_keys = {}
-        property_dict = {}
-        deleted_keys = []
-        for sort in sorting_list:
-            for prop_name in sort.get_property_keys():
-                if prop_name in deleted_keys:
-                    continue
-                if prop_name in property_keys:
-                    if property_keys[prop_name] != sort.get_property(prop_name).dtype:
-                        print(f"Skipping property '{prop_name}: difference in dtype between sortings'")
-                        del property_keys[prop_name]
-                        deleted_keys.append(prop_name)
-                else:
-                    property_keys[prop_name] = sort.get_property(prop_name).dtype
-        for prop_name in property_keys:
-            dtype = property_keys[prop_name]
-            property_dict[prop_name] = np.array([], dtype=dtype)
+        # Check if all the sortings have the same properties
+        properties_set = set(np.concatenate([sorting.get_property_keys() for sorting in sorting_list]))
```

Comment:
Nice.
Better concatenate that use the ugly double iteration notation of python lists:

```python
all_books = [book for shelf in bookstore for book in shelf]
```